transform/transformers/pck_transformer.py

```diff
@@ -134,13 +135,21 @@ def populate_period_data(self):
                 self.data['12'] = end_date.strftime("%d/%m/%Y")
 
     def round_currency_values(self):
-        """For RSI/QCAS Surveys, round the values of the currency fields.
+        """For RSI Surveys, round the values of the currency fields.
         Rounds up if the value is .5
+
+        For QCAS Surveys, round the values of the currency fields and divide
+        by 1000 (i.e., 56100 would return 56)
         """
-        if self.survey.get('survey_id') in [self.rsi_survey_id, self.qcas_survey_id]:
+        if self.survey.get('survey_id') in [self.rsi_survey_id]:
             self.data.update({k: str(Decimal(v).quantize(Decimal('1.'), ROUND_HALF_UP))
```

Comment:
Since for the code below you are calling a method, I think its worth putting this into its own method also called round_to_nearest_unit?

Reply:
I disagree, both of these are rounding currency values.  They're just doing it differently for different surveys.  Until we do some work to tidy up the survey specific stuff that's crept its way into this generic module, I think having a high level function that describes what it's doing is better then having 2 smaller ones that describe survey specific stuff

Reply:
The method below doesn't have any knowledge of what survey it is handling...It is purely the logic for rounding, which isn't survey specific. I just mean the one has the code of rounding within the method, the other call a method to do another type of rounding. Its just nice to be consistent.

Reply:
Up to you. No big deal, I just rather have both sets of code either inline or both in the method for consistency. Just preference I guess.